src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/base.html.twig

```diff
@@ -24,20 +24,48 @@
   {% endjavascripts %}
 </head>
 <body id="installer">
-<div id="genericBox" class="container">
-  <div class="row">
-    <div class="col-md-10 col-md-offset-1">
-      <div id="genericBoxTitle" class="page-header text-center">
-        <h2>Install Fork CMS</h2>
+<div id="root">
+  <div id="genericBox">
```

Comment:
I believe the genericBox id's are useless, they're not used in any CSS or JS files